docs/_includes/floating-label.html

```diff
@@ -322,6 +322,160 @@ <h3>Floating Label with large select</h3>
 </span>
     {% endhighlight %}
 
+    <h3>Floating Label with Phone input</h3>
+    <p>For phone input, the label always remains in floating state above the input. This state ensures that the label does not obscure the phone number.</p>    
+    <div class="demo">
+      <div class="demo__inner">
+        <span class="phone-input">
+          <span class="listbox-button listbox-button--form">
+            <button class="btn btn--form btn--split-start" aria-expanded="false" aria-haspopup="listbox" type="button"
+              data-listbox-button-prefix="Country: " aria-label="Country: United States of America (+1)">
+              <span class="btn__cell">
+                <span class="btn__text">
+                  <svg class="flag flag--us" focusable="false" height="18" width="24" aria-hidden="true" data-country-code="1">
```

Comment:
If `height` and `width` are not included, this breaks. We should make sure that in phone input we are targetting `.phone-input listbox-button btn svg.flag` and setting height and width on that.